packages/blue-sdk/src/types.ts

```diff
@@ -1,14 +1,15 @@
-export type Hex64 = string & { __LENGTH__: 64 };
+export type Hex<L extends number = -1> = `0x${string}` &
+  (L extends -1 ? unknown : { __LENGTH__: L });
 
 /**
  * The address of a Contract, or an EOA
  */
-export type Address = string;
+export type Address = Hex;
```

Comment:
```suggestion
export type Address = `0x{string}`;
```

A reviewer could be tempted to move to `Hex<20>` otherwise (semantically Hex<20> = Address < Hex)

Reply:
IMO we should get rid of the `Hex`